server/api.js

```diff
@@ -1,10 +1,133 @@
-'use strict'
-const api = require('express').Router()
-const db = require('../db')
+'use strict';
+const api = require('express').Router();
+const db = require('../db');
+const { Student, Campus } = require('../db/models/');
 
+// console.log('api : module: ', module);
+// console.log('api : require.main: ', require.main);
 // If you aren't getting to this object, but rather the index.html (something with a joke) your path is wrong.
-	// I know this because we automatically send index.html for all requests that don't make sense in our backend.
-	// Ideally you would have something to handle this, so if you have time try that out!
-api.get('/hello', (req, res) => res.send({hello: 'world'}))
+// I know this because we automatically send index.html for all requests that don't make sense in our backend.
+// Ideally you would have something to handle this, so if you have time try that out!
+api.get('/hello', (req, res) => res.send({ hello: 'world' }))
 
-module.exports = api
\ No newline at end of file
+api.get('/students', (req, res, next) => {
+	Student.findAll({include: [Campus]})
```

Comment:
Your editor likely has a feature to automated converting tabs <-> spaces as needed if you end up with a mis-match.